packages/amplify_core_plugin_interface/lib/amplify_datastore_category.dart

```diff
@@ -47,4 +47,10 @@ class DataStoreCategory {
             where: where, pagination: pagination, sortBy: sortBy)
         : throw (errorMsg);
   }
+
+  Future<T> deleteInstance<T extends Model>(T model, {QueryPredicate when}) {

```

Comment:
Did we decide to do just `delete`, instead of `deleteInstance`?

Reply:
Ah right, I fixed this.